### ISSUE TITLE
Add players to chunk loaders

### ISSUE THREAD

**Issue by Shybella** [TheCBProject/ChickenChunks#29]
Hello, 

I've had numerous requests from my players to have the ability to add people to their chunk loaders. The reason this request comes up so much is I have the chunk-loaders only active while the player is online. Players usually group up and play together but if the player logs off so does that chunk-loader. It seems redundant and inefficient to have multiple chunk-loaders in one chunk due to a player logging off.

tl;dr Ability to add players to chunk loaders to keep them active while chunk-loader owner is offline.

Thank you.

**Comment by Shybella** [comment]
This feature could be added via a plugin using the SpongeAPI but this has been done before and has some problems. This mod is the only successful Chunk-loading mod that I see as being practical. This isn't a required feature but WOULD be greatly appreciated.

**Comment by covers1624** [comment]
I really like the idea, CC @Chicken-Bones, thoughts?

**Comment by Shybella** [comment]
Thank you for the response, cover1624. 

One issue I'm running into this idea with is you can just assign any name you want to the chunk-loader because I could add any active player to keep my chunks loaded even while offline, this is a problem.

The idea would be to force the player that's being added to that specific chunk-loader to punch that specific chunk-loader so no one can randomly add any random active player. 

Or maybe request access somehow.

Thank you again guys

**Comment by Chicken-Bones** [comment]
Good idea. Make it that if they place a chunkloader on an existing one, it gets consumed inside and their name gets added? If they break it they only get theirs back?

Just an implementation thought.

Also the lasers need a visual update :p

**Comment by Shybella** [comment]
I like your idea way more it makes sense.

**Comment by Chicken-Bones** [comment]
Could even make the number of floating pearls match the number of players, forming a kind of ring.

**Comment by covers1624** [comment]
Sounds good, ill work on this for the 1.13 port (when that happens), or perhaps just before it.

**Comment by Shybella** [comment]
Thank you for taking the time and adding this feature. It'll be really useful..

This will be a 1.13 feature or will be back ported to 1.12.2?

**Comment by covers1624** [comment]
1.13 is such a long way away, ill likely get to this before then. My list is quite large atm..

**Comment by Shybella** [comment]
Great! Thank you again.
